CIP-1694/README.md

```diff
@@ -509,20 +508,36 @@ for example, a motion of no confidence is enacted.
 | 6. Treasury Withdrawals                                             | Movements from the treasury, sub-categorized into small, medium or large withdrawals (based on the amount of Lovelace to be withdrawn). The thresholds for treasury withdrawals are discussed below. |
 | 7. Info                                                             | An action that has no effect on-chain, other than an on-chain record. |
 
-**Any Ada holder** can submit a governance action to the chain.
-They must provide a deposit of `govDeposit` Lovelace, which will be returned when the action is finalized
-(whether it is **ratified**, has been **dropped**, or has **expired**).
-The deposit amount will be added to the _deposit pot_, similar to stake key deposits.
-It will also be counted towards the stake of the reward address it will be paid back to, to not reduce the submitter's voting power to vote on their own (and competing) actions.
-
-Note that a motion of no-confidence is an extreme measure that enables Ada holders to revoke the power
-that has been granted to the current constitutional committee.
-Any outstanding governance actions, including ones that the committee has ratified or ones that would be enacted this epoch,
-will be dropped if the motion is enacted.
+> **Note**
+> motion of no-confidence is an extreme measure that enables Ada holders to revoke the power
+> that has been granted to the current constitutional committee.
+> Any outstanding governance actions, including ones that the committee has ratified or ones that would be enacted this epoch,
+> will be dropped if the motion is enacted.
 
 > **Note**
 > A **single** governance action might contain **multiple** protocol parameter updates. Many parameters are inter-connected and might require moving in lockstep.
 
+#### Fee
+
+**Any Ada holder** can submit a governance action to the chain.
+Transactions that carry governance actions must satisfy a **minimum fee** to be valid.
+The minimum fee is parameterized by a new protocol parameter `minGovernanceFee`.
+
+When a governance action is ratified, or when it expires,
+any governance fee surplus is distributed amongst DReps **that took part in the vote**, regardless of their vote.
```

Comment:
```suggestion
any governance fee surplus is distributed amongst registered DReps **that took part in the vote**, regardless of their vote.
```
To avoid confusion with pre-defined DReps?

Reply:
Using ''registered'' DRep could possibly conflict with the changes proposed by #545 about DRep registration if merged. 
Suggestion:  Maybe ''active DReps'' instead ?